.github/workflows/testing.yml

```diff
@@ -73,6 +73,9 @@ jobs:
       - name: Run more tests
         run: poetry run python sycamore/tests/manual/test_fast_sycamore_import.py
         working-directory: lib/sycamore
+      - name: Run query-ui test
+        run: poetry run bash -c 'cd ../../apps/query-ui && PYTHONPATH=. pytest .'
+        working-directory: lib/sycamore
```

Comment:
Why not just set `working-directory` to `apps/query-ui` and then do `poetry run pytest .` ?

Reply:
Because the poetry environment is in lib/sycamore.  Look at the install-sycamore step above.